src/test/java/prgms/vouchermanagementapp/repository/JdbcVouchersTest.java

```diff
@@ -0,0 +1,111 @@
+package prgms.vouchermanagementapp.repository;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import prgms.vouchermanagementapp.domain.Customer;
+import prgms.vouchermanagementapp.domain.Voucher;
+import prgms.vouchermanagementapp.domain.VoucherEntity;
+import prgms.vouchermanagementapp.domain.value.Amount;
+import prgms.vouchermanagementapp.domain.value.Ratio;
+import prgms.vouchermanagementapp.repository.util.EntityMapper;
+import prgms.vouchermanagementapp.service.CustomerManager;
+import prgms.vouchermanagementapp.service.VoucherCreationFactory;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+class JdbcVouchersTest {
+
+    @Autowired
+    private CustomerManager customerManager;
+
+    @Autowired
+    private JdbcVouchers jdbcVouchers;
+
+    private Customer savedCustomer;
+
+    @BeforeEach
+    void beforeEach() {
+        savedCustomer = customerManager.save(new Customer("customer"));
+    }
+
+    @DisplayName("고객에게 할당된 바우처를 저장할 수 있다.")
+    @Test
+    void save() {
+        // given
+        Voucher voucher1 = VoucherCreationFactory.createVoucher(new Amount(3000));
+        Voucher voucher2 = VoucherCreationFactory.createVoucher(new Ratio(50));
+        VoucherEntity voucherEntity1 = EntityMapper.toVoucher(voucher1, savedCustomer);
+        VoucherEntity voucherEntity2 = EntityMapper.toVoucher(voucher2, savedCustomer);
+
+        // when
+        jdbcVouchers.save(voucherEntity1);
+        jdbcVouchers.save(voucherEntity2);
+        List<VoucherEntity> voucherEntities = jdbcVouchers.findAll();
+
+        // then
+        assertThat(voucherEntities.size()).isEqualTo(2);
+    }
+
+    @DisplayName("고객이 가진 고정 금액 할인 바우처의 금액을 변경할 수 있다.")
+    @Test
+    void updateFixedAmountVoucherById() {
+        // given
+        long originalAmount = 1000;
+        Voucher originalVoucher = VoucherCreationFactory.createVoucher(new Amount(originalAmount));
+        VoucherEntity originalVoucherEntity = EntityMapper.toVoucher(originalVoucher, savedCustomer);
+        jdbcVouchers.save(originalVoucherEntity);
+
+        // when
+        long changedAmount = 2000;
+        jdbcVouchers.updateFixedAmountVoucherById(originalVoucherEntity.getId(), changedAmount);
+        Optional<VoucherEntity> foundVoucherEntity = jdbcVouchers.findVoucherEntityById(originalVoucherEntity.getId());
+        assert foundVoucherEntity.isPresent();
+
+        // then
+        assertThat(foundVoucherEntity.get().getAmount()).isEqualTo(changedAmount);
```

Comment:
여기 테스트가 조금 복잡하네요.
결과값은 미리 추출해서 assert문에서는 

```suggestion
        assertThat(actual).isEqualTo(2000);
```
이런식으로 표현하는게 읽기 좋더라구요.